### ISSUE TITLE
Analysis Time Series.py

### ISSUE THREAD

**Issue by LePetitNewbie** [michaelmarzec/DFS-FanDuel-Optimization#1]
Hi Michael
            I tried running these codes as I think this looks great. I did run the Optimization.py (worked) and the Game_Log Scraper also did the trick. I do not have any coding experience and encountered lots of red messages when running the AnalysisTimeSeries.py. Below is the error I get. If you have any advice for me to get it to work please do. anyways thanks in advance if you can help. Take care

![image](https://user-images.githubusercontent.com/81279182/146318541-03d92cfa-2215-4a97-bd88-4a6689b4e8de.png)

Ol

**Comment by michaelmarzec** [comment]
Hello - thanks for reaching out. There are likely upstream effects occurring within the data processing that are causing the time-series analysis to fail. If you are looking to enhance this work (or learn via exploration) I would encourage you to look at the inputs to the AnalysisTimeSeries and/or research the error messages -- the ValueError in the last line "expected 4, got 3" likely suggests that the player it's attempting to predict for does not have enough games played and thus a filtering step has been missed. 

If instead, you are looking to use the scripts outputs, I am currently working on an update to better operationalize this model which I am not yet ready to share, but will upload once it's finished! 

Hope this helps and please feel free to reach out with any other questions.

**Comment by LePetitNewbie** [comment]
Hello - thanks for your reply. To be honest, this is out of my depth. I tried running the scripts again. I started with the game_log data scraping. I moved the csv file to the Analysis time series folder. I get this error with the first player (aaron gordon) and by the time it gets to aleksej pokusevski, it stops completely. I did give it a go - but I do not understand what this means. I am looking for the output but still fascinated by how it gets there. I am not really using it for fanduel as I am from Australia (but the way points are awarded in the game here is pretty similar)... but this part I can change the salary and the rules in the optimization.py (this script works)..anyways, i am not sure really but i'll wait  for your update. 
Cheers mate
Ol
